frontend/src/queries/nodes/InsightViz/TrendsSeries.tsx

```diff
@@ -90,6 +90,7 @@ export function TrendsSeries(): JSX.Element | null {
                     TaxonomicFilterGroupType.Actions,
                     TaxonomicFilterGroupType.DataWarehouse,
                 ]}
+                hideDeleteBtn={actionsAndEventsCount(filters as any) === 1}
```

Comment:
Would be nice to base this on the query already (we need to convert to filters here as the `<ActionFilter />` component doesn't support queries yet.

```suggestion
                hideDeleteBtn={series?.length === 1}
```

Add a selector for `series` to the selectors from `insightVizDataLogic` in line 22 above.